### PR TITLE
Update variable declaration

### PR DESCRIPTION
**Pull request by Thesaleem**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Thesaleem** [comment]
Hi @TildaDares! Kindly review the opened PR.

File: src/DistortableImageOverlay.js [hunk 1]
```diff
@@ -228,13 +228,13 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
...
-    var edit = this.editing;
-    var i = 0;
+    const map = this._map;
+    const zoom = map.getZoom();
+    const edit = this.editing;
+    let i = 0;
 
     // this is to fix https://github.com/publiclab/Leaflet.DistortableImage/issues/402
-    for (var k in latlngObj) {
+    for (let k in latlngObj) {
```
Comment (TildaDares) [inline src/DistortableImageOverlay.js:237]: Since we use `let` here, we need to also declare `k` with `let` on line 246.

Same thing applies to line 280

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: src/DistortableImageOverlay.js (hunk 1 above)
Comment (Thesaleem) [reply src/DistortableImageOverlay.js:237]: Okay, i'll get it done

Comment (Thesaleem) [reply src/DistortableImageOverlay.js:237]: Done! @TildaDares

**Review by TildaDares — APPROVED** [review]
Amazing work @Thesaleem. Thank you!!

**Comment by Thesaleem** [comment]
Thank youuuu! 🥳

**Pull request merged**